### PR TITLE
nginx: update brotli module

### PR DESCRIPTION
**Pull request by dengqf6**
Maintainer: @Ansuel 
Compile tested: mvebu
Run tested: mvebu, WRT1900ACv2

Description:
Update brotli module to the latest git head
Adjust conf files

### CODE REVIEW

**Comment by Ansuel** [comment]
Thx a lot

File: net/nginx/Makefile [hunk 1]
```diff
@@ -429,18 +429,18 @@ endif
 
 ifeq ($(CONFIG_NGINX_HTTP_BROTLI),y)
   define Download/nginx-brotli
-    VERSION:=e26248ee361c04e25f581b92b85d95681bdffb39
+    VERSION:=dc37f658ccb5a51d090dc09d1a2aca2f24309869
     SUBDIR:=nginx-brotli
-    FILE:=ngx-brotli-module-$$(VERSION).tar.gz
+    FILE:=ngx-brotli-module-$$(VERSION).tar.xz
     URL:=https://github.com/eustas/ngx_brotli.git
```
Comment (neheb) [inline net/nginx/Makefile]: .git at the end can be removed.

File: net/nginx/Makefile [hunk 2]
```diff
@@ -429,18 +429,18 @@ endif
...
-    MIRROR_HASH:=76b891ba49f82f0cfbc9cba875646e26ee986b522373e0aa2698a9923a4adcdb
+    MIRROR_HASH:=6bc0c40ff24f6e0ac616dfddc803bdc7fcf54764ba9dc4f9cecb3a68beedcdaf
     PROTO:=git
   endef
   $(eval $(call Download,nginx-brotli))
 
   define Prepare/nginx-brotli
 	$(eval $(Download/nginx-brotli))
-	gzip -dc $(DL_DIR)/$(FILE) | tar -C $(PKG_BUILD_DIR) $(TAR_OPTIONS)
+	xz -dc $(DL_DIR)/$(FILE) | tar -C $(PKG_BUILD_DIR) $(TAR_OPTIONS)
```
Comment (neheb) [inline net/nginx/Makefile]: The default in download.mk is -7e. Not sure if we should match here.

Comment (dengqf6) [reply net/nginx/Makefile]: `-7e` is *compression* level, this is *decompression*

Comment (neheb) [reply net/nginx/Makefile]: Ah my mistake.

**Comment by diizzyy** [comment]
This should  be made into two commits

File: net/nginx/Makefile (hunk 1 above)
Comment (dengqf6) [reply net/nginx/Makefile]: I second @jefferyto's opinion in #9626, the `.git` postfix is a standard and we'd better keep it.

**Pull request merged**